File: apple/OmnivoreKit/Sources/Models/NewsletterEmail.swift
import Foundation

public struct NewsletterEmail {
  public let id: String
  public let email: String
  public let confirmationCode: String?

  public init(id: String, email: String, confirmationCode: String?) {
    self.id = id
    self.email = email
    self.confirmationCode = confirmationCode
  }
}

File: android/Omnivore/app/src/main/java/app/omnivore/omnivore/persistence/entities/NewsletterEmail.kt
package app.omnivore.omnivore.persistence.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class NewsletterEmail(
  @PrimaryKey val userID: String,
  val confirmationCode: String?,
  val email: String?,
  val emailID: String?
)
